scripts/update-version.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const RELAY_NPM_PACKAGES = [
+  'babel-plugin-relay',
+  'react-relay',
+  'relay-compiler',
+  'relay-runtime',
+  'relay-test-utils',
+  'relay-test-utils-internal',
+];
+
+const CARGO_FILES = [
```

Comment:
It's missing `const COMPILER_TARGETS_FILE = `${ROOT_DIR}/fbcode/relay/oss/crates/relay-compiler/TARGETS`;` from the original script, but I'm not sure if a problem.